components/launcher-client/src/client.rs

```diff
@@ -40,7 +52,11 @@ impl LauncherCli {
         Self::send(&tx, &cmd)?;
         let (rx, raw) = ipc_srv.accept().map_err(|_| Error::AcceptConn)?;
         Self::read::<protocol::NetOk>(&raw)?;
-        Ok(LauncherCli { tx: tx, rx: rx })
+        Ok(LauncherCli {
```

Comment:
It might be good to name the incoming `pipe` variable and the `pipe` variable returned by the `IpcServer::new()` call differently to avoid potential confusion with exactly which thing we're storing.